ts/src/htx.ts

```diff
@@ -6455,12 +6455,7 @@ export default class htx extends Exchange {
         if (ecid !== undefined) {
             request['chain'] = this.ecidToNetworkId (ecid, code);
         }
-        const networks = this.safeValue (currency, 'networks', {});
-        const networkItem = this.safeValue (networks, ecid, {});
-        let precision = this.safeValue (currency, 'precision');
-        precision = this.safeString (networkItem, 'precision', precision);
-        precision = this.precisionFromString (String (precision));
-        amount = parseFloat (this.decimalToPrecision (amount, TRUNCATE, precision));
+        amount = this.currencyToPrecision (code, amount, ecid);
```

Comment:
Did you check the precision format in networks of currencies? Maybe they've used `precisionFromString` for some reasons

Reply:
Original code
```
        const networks = this.safeValue (currency, 'networks', {});
        const networkItem = this.safeValue (networks, ecid, {});
        let precision = this.safeValue (currency, 'precision');
        precision = this.safeString (networkItem, 'precision', precision);
        precision = this.precisionFromString (String (precision));
        amount = parseFloat (this.decimalToPrecision (amount, TRUNCATE, precision));
```

New code
```
        amount = this.currencyToPrecision (code, amount, ecid);

-> currencyToPrecision
            let precision = ex.safeValue(currency, 'precision');     // <=> 6460
            if (networkCode !== undefined) {
                    const networks = ex.safeDict(currency, 'networks', {}); // <=> 6458
                    const networkItem = ex.safeDict(networks, networkCode, {}); // <=> 6459
                    precision = ex.safeValue(networkItem, 'precision', precision); // <=> 6461
                    console.log("precision", precision)
                }
               ...
               return this.decimalToPrecision (fee, TRUNCATE, precision, this.precisionMode, this.paddingMode);

-> decimalToPrecision
                  ...
    if (countingMode === TICK_SIZE) { // It is the case
         ...
        const newNumPrecisionDigits = precisionFromString(precisionDigitsString); // <=> 6462
        ...
        return decimalToPrecision(x, roundingMode, newNumPrecisionDigits, DECIMAL_PLACES, paddingMode); <=> 6363
    }
```

Reply:
Alright, thanks!